dspace-api/src/main/java/org/dspace/curate/CurationCli.java

```diff
@@ -147,9 +152,17 @@ public static void main(String[] args) throws Exception {
         }
 
         Curator curator = new Curator();
-        if (reporterName != null) {
-            curator.setReporter(reporterName);
+        OutputStream reporter;
+        if (null == reporterName) {
```

Comment:
Update the CLI usage information to describe the values that may be passed in for reporter.  

Clarify that a relative path name or absolute path name can be specified.  Clarify the intention of the "-" value.

In my testing, output seemed to be written to the log as well as to the file.  Should curation output be suppressed from the log when a ReportWriter is active?

Reply:
If a task calls e.g. Logger.info() then that goes straight to the log.  If a task calls Curator.report() then that should go straight to the Reporter that was set and not to the log.  A task may have different things to say in the log and in a report.  Of course a task may send the same text out both channels, separately.